CHANGELOG.rst

```diff
@@ -5,13 +5,29 @@ Community Sops Release Notes
 .. contents:: Topics
 
 
+v1.0.2
+======
+
+Release Summary
+---------------
+
+Fix of 1.0.1 release which had no changelog entry.
+
+v1.0.1
+======
+
+Release Summary
+---------------
+
+Re-release of 1.0.0 to counteract error during release.
+
 v1.0.0
 ======
 
 Release Summary
 ---------------
 
-First stable release. This release is expected to be included in Ansible 3.0.0.
+First stable release. This release is expected to be included in Ansible 2.11.
```

Comment:
Not a blocker, but why is this back to `2.11`? It was my understanding that our target was `3.0.0`. Or the target changed because this collection `1.0.0` was already in `2.11`?

Reply:
Probably because changelogs/changelog.yaml says to. That's the source of truth out of which this file is created.